R/mod_tabpanel_data.R

```diff
@@ -62,8 +62,8 @@ mod_tabpanel_data_ui <- function(id) {
           htmlOutput(ns("unused_vars"))
         ),
         hr(),
-        h4("Cases which have missing data"),
-        span(paste0("In this section you receive feedback about cases for which data required for the computations is missing and will thus be removed for all analysis. The following variables are checked for missingness: ",check_for_missing_values(), ". If any of these variables have a missing value the case is removed for all analysis.")),
+        h4("Data Quality"),
+        span(paste0("In this section you receive feedback about data quality and cases for which data required for the computations is missing and will thus be removed for all analysis. The following variables are checked for missingness: ",check_for_missing_values(), ". If any of these variables have a missing value the case is removed for all analysis.")),
```

Comment:
Maybe we can change the text to the following which is a bit easier to read:
"In this section you receive feedback about implausible or inconsistent values and missing data. The following variables are checked for missingness: ",check_for_missing_values(), ". The following variables are checked for inconsistent/implausible values: age.

Reply:
I agree, what you are suggesting is better!